demo/Snakefile

```diff
@@ -46,12 +47,13 @@ config["lcr-modules"]["_shared"]["samples"] = SAMPLES
 # Load module-specific snakefiles
 include: "../modules/star/1.0/star.smk"
 include: "../modules/manta/2.0/manta.smk"
-
+include: "../modules/strelka/1.0/strelka.smk"
```

Comment:
Just in case we want to add a strelka1 module later, could you make sure the module name is `strelka2` everywhere? Thanks!